src/stories/Card.stories.ts

```diff
@@ -0,0 +1,24 @@
+import type { Meta } from "@storybook/react";
+import Card from "../components/Common/Card/Card";
+
+const meta = {
+  title: "Example/Card",
+  component: Card,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+} satisfies Meta<typeof Card>;
+
+export default meta;
+
+export const Default = {
```

Comment:
Try to show different variations of the Card.
Will explain this in the standup.